Makefile

```diff
@@ -25,9 +25,17 @@ env:: | $(CONDA_ENV_PYTHON)
 	git submodule init
 	git submodule update --init --recursive
 	mkdir -p env/symbiflow
+	wget -q https://github.com/QuickLogic-Corp/quicklogic-fpga-toolchain/releases/download/v1.1.0/Symbiflow_v1.1.0.gz.run
+	mkdir env/quicklogic
+	chmod +x Symbiflow_v1.1.0.gz.run
+	INSTALL_DIR=${PWD}/env/quicklogic ./Symbiflow_v1.1.0.gz.run
+	rm Symbiflow_v1.1.0.gz.run
+	source ./env/quicklogic/conda/bin/activate && pip3 install asciitable terminaltables colorclass "git+https://github.com/antmicro/edalize.git@quicklogic#egg=edalize"
```

Comment:
There should be only one `&` otherwise all that pip stuff gets installed outside the env.